tutorials/postman-api/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+meta:
+  title: How to use Postman to manage Scaleway APIs
+  description: This page shows how to import APIs into Postman, configure your environment variables and authentication and how to share your workspace with other users. 
+content:
+  h1: Managing Scaleway APIs with Postman
+  paragraph: This page shows how to import Scaleway APIs into Postman, configure your environment variables and authentication and how to share your workspace with other users.  
+tags: postman postman-api scaleway-api postman-download request json
+categories:
+  - storage
```

Comment:
This applies to all Scaleway products, `storage` probably is not the right category here.